### PR TITLE
Adding git steps to MFA Enforcement page

### PR DESCRIPTION
**Pull request by billygilbert**
## Purpose of this pull request

This pull request (PR) adds references to Git to MFA documentation.  Also corrects errors about token authentication.

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/project/project-enable-mfa-enforcement.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/project/project-enable-mfa-enforcement.md [hunk 1]
```diff
@@ -91,7 +88,46 @@ To connect using SSH with 2FA user account credentials:
 
    web@mymagento.0:~$
    ```
+   
+   
+## Clone code using SSH with 2FA
+
+Cloud projects authenticate to the Git repository over SSH.  Projects with MFA enabled require an SSH certificate to be generated to perform operations on the Git repository.
```
Comment (meker12) [inline src/cloud/project/project-enable-mfa-enforcement.md:95]:
```suggestion
When managing source code for {{site.data.var.ece}} projects, you use SSH to authenticate to the Git repository for the project.  If your project has  MFA enforcement enabled, you must generate an SSH certificate before you can perform command line operations using the Git repository.
```

File: src/cloud/project/project-enable-mfa-enforcement.md [hunk 2]
```diff
@@ -91,7 +88,46 @@ To connect using SSH with 2FA user account credentials:
...
+ 
+{:.procedure}
+To connect using SSH with 2FA user account credentials:
+
+1. Log in to your {{site.data.var.ece}} account and authenticate using 2FA.
```
Comment (meker12) [inline src/cloud/project/project-enable-mfa-enforcement.md:100]:
```suggestion
1. Log in to your {{site.data.var.ece}} account and authenticate using 2FA.

   {:.bs-callout-info}
   If you do not have 2FA enabled on your account, you must enable it. See [Enable 2FA on {{ site.data.var.ece }} accounts]({{ site.baseurl }}/cloud/project/user-admin.html#enable-2fa-for-cloud-accounts).
   
```

File: src/cloud/project/project-enable-mfa-enforcement.md [hunk 3]
```diff
@@ -91,7 +88,46 @@ To connect using SSH with 2FA user account credentials:
...
+     Expires at: 2020-07-13T15:28:13-04:00
+     Multi-factor authentication: verified
+     Mode: interactive
+   The certificate will be automatically refreshed when necessary.
+   Checking SSH configuration file: /Users/<user-name>/.ssh/config
+   Do you want to update the file automatically? [Y/n] Y
+   Configuration file updated successfully: /Users/<user-name>/.ssh/config
+   ```
+   
+1. Clone the Git repository of your project:
```
Comment (meker12) [inline src/cloud/project/project-enable-mfa-enforcement.md:119]:
```suggestion
1. Clone the Git repository for your project environment:
```

File: src/cloud/project/project-enable-mfa-enforcement.md [hunk 4]
```diff
@@ -34,9 +34,6 @@ magento-cloud ssh-cert:load
 
 The `ssh-cert:load` command generates the SSH certificate and installs it in the SSH agent of the local user.
 
-{:.bs-callout-tip}
```
Comment (meker12) [inline src/cloud/project/project-enable-mfa-enforcement.md:37]: Did you remove this on purpose?

**Comment by billygilbert** [comment]
Setting as onhold as there are a few more updates I would like to add.

File: src/cloud/project/project-enable-mfa-enforcement.md (hunk 4 above)
Comment (billygilbert) [reply src/cloud/project/project-enable-mfa-enforcement.md:37]: Yes, it is inaccurate as written.  Even with an api token generate, you still need to generate an SSH certificate.

File: src/cloud/project/project-enable-mfa-enforcement.md [hunk 5]
```diff
@@ -143,7 +179,7 @@ To authenticate and reload the SSH certificate:
 
 When MFA enforcement is enabled on a {{site.data.var.ece}} project, automated processes that require SSH access to a Cloud environment must authenticate using an API token.
 
-The API token allows the SSH connection request to bypass the user authentication flow that requires a 2FA token and SSH certificate. You generate the token from a {{site.data.var.ece}} account with Admin or Contributor access on the project.
```
Comment (meker12) [inline src/cloud/project/project-enable-mfa-enforcement.md:146]: Perhaps this should have at the end -- You must generate the token after logging  into the account using 2FA credentials.

Comment (billygilbert) [reply src/cloud/project/project-enable-mfa-enforcement.md:146]: Sure, will add that portion back.

**Comment by meker12** [comment]
running tests

**Pull request merged**